tao_compiler/mlir/xla/ral/context/common_context_impl_cuda.cc

```diff
@@ -84,9 +86,19 @@ static inline buffer_t ral_base_cuda_const_cuda_internal(
         ctx->signalError(Context::FAILURE, msg);
       }
       std::string hex_str = constants.at(key);
-      TAO_VLOG(2) << "fromHex start:";
       auto data = fromHex(hex_str);
       auto bytes = data.size();
+      int64_t num_elements = std::accumulate(dim_sizes.begin(), dim_sizes.end(),
+                                             1, std::multiplies<int64_t>());
+      if (bytes < num_elements * width_in_bytes) {
+        // isSplat
+        bytes = num_elements * width_in_bytes;
+        auto splat_data = data;
+        for (int64_t i = 0; i < num_elements - 1; ++i) {
```

Comment:
NIT: maybe we could use memset on GPU？ For large const，it may be much faster.

Reply:
memset api is only able to handle 32bit set, which is not enough.